integration-cli/docker_cli_run_unix_test.go

```diff
@@ -299,7 +299,10 @@ func (s *DockerSuite) TestRunEchoStdoutWithMemoryLimit(c *check.C) {
 	}
 }
 
-// should run without memory swap
+// TestRunWithoutMemoryswapLimit should run without memory swap limit.
```

Comment:
The previous comment is misleading.
The right comment should be "should run without memory swap limit", not "should run without memory swap".
In my opinion, "without memory swap" is meaning that no memory swap can be used.